### ISSUE TITLE
haml-lint: invalid option: --no-summary

### ISSUE THREAD

**Issue by Br3nda** [brigade/overcommit#486]
overcommit is saying
```
Analyze with haml-lint.....................................[HamlLint] FAILED
Unexpected output: unable to determine line number or type of error/warning for output:
invalid option: --no-summary
Run `haml-lint` --help for usage documentation
```
see here: https://travis-ci.org/Br3nda/growstuff/jobs/221583675

but our haml-lint config doesn't use `--no-summary`
```
  HamlLint:
    enabled: true
    requires_files: true
    on_warn: fail
    command: ['bundle', 'exec', 'haml-lint', 'app/views']
```
(whole file here) https://github.com/Br3nda/growstuff/blob/dev/.overcommit.yml

Any idea where that --no-summary comes from?

**Comment by Br3nda** [comment]
this was working -- but broken when haml-lint was updated recently

**Comment by trotzig** [comment]
Sorry for the inconvenience @Br3nda. The flag comes from the default config for `HamlLint` in overcommit and it was added after we pushed [v0.22.0 of Haml-lint](https://github.com/brigade/haml-lint/releases/tag/v0.22.0). I think there are two potential fixes for this problem: 
- Add `flags: []` to your local overcommit config for haml-lint to override the defaults
- Upgrade haml-lint in your project. 

The first option is more lightweight, but I'm not 100% sure it will work.
